installers/python-ci/radiasoft-download.sh

```diff
@@ -43,10 +44,11 @@ python_ci_main() {
             set -x
             cd '$d'
             export GITHUB_TOKEN='${GITHUB_TOKEN:-}'
-            if (( $p )); then
-                pip uninstall -y pykern >& /dev/null || true
-                pip install git+https://github.com/radiasoft/pykern.git
-            fi
+            for x in "${p[*]}"
```

Comment:
Two syntax issues:

`for x in ${p[*]}; do`

The quotes are unnecessary, and I believe will cause issues. This assumes $p values do not contain spaces or specials (which is true), but it should be noted.

Reply:
085af5b7bc7474e434c3f7ad7cc88ba32ed8cbb8